kodi: update to 19.0RC1

HiassofT opened the pull request:
runtime tested on RPi4

HiassofT commented:
RPi2 tested fine, too

heitbaum commented:
Generic tested
```
=== tested on ===
 14:50:12 up 14 min,  load average: 0.15, 0.56, 0.53
Generic.x86_64-devel-20210116004441-792d993
Linux LibreELEC 5.10.7 #1 SMP Thu Jan 14 14:16:10 UTC 2021 x86_64 GNU/Linux
Starting Kodi (19.0-RC1 (18.9.910) Git:19.0RC1-Matrix). Platform: Linux x86 64-bit
Using Release Kodi x64
Kodi compiled 2021-01-16 by GCC 10.2.0 for Linux x86 64-bit version 5.10.7 (330247)
Running on LibreELEC (community): devel-20210116004441-792d993 9.80, kernel: Linux x86 64-bit version 5.10.7
FFmpeg version/source: 4.3.1-Kodi
Host CPU: Intel(R) Core(TM) i5-6260U CPU @ 1.80GHz, 4 cores available
[    0.000000] DMI:  /NUC6i5SYB, BIOS SYSKLi35.86A.0073.2020.0909.1625 09/09/2020
CRenderSystemGL::InitRenderSystem - Version: 4.6 (Core Profile) Mesa 20.3.3, Major: 4, Minor: 6
GL_RENDERER = Mesa Intel(R) Iris(R) Graphics 540 (SKL GT3)
GL_VERSION = 4.6 (Core Profile) Mesa 20.3.3
```

HiassofT commented:
I manually bumped visualization.matrix to latest githash in Matrix branch, addon version was bumped to 0.5.0 in October but release/tag is missing - see https://github.com/xbmc/visualization.matrix/issues/5

Also added vfs.sacd to bumpscript exclude list - we should remove that at some point as it has been deprecated and removed from kodi binary repo some time ago
